sdks/java/io/kinesis/build.gradle

```diff
@@ -32,9 +32,9 @@ test {
 def aws_version = "1.11.255"
 
 dependencies {
-  compile library.java.guava
   compileOnly library.java.findbugs_annotations
   shadow project(path: ":beam-sdks-java-core", configuration: "shadow")
+  shadow library.java.guava
```

Comment:
Should we override the default shadow closure as well otherwise we'll get one repackaged guava and also say we depend on regular guava?

Reply:
Actually nevermind, the compile scope doesn't have the dependency so it won't be shaded.